Tests/TestHelpers.swift

```diff
@@ -1,12 +1,12 @@
 import Moya
-import Foundation
 
-extension String {
-    var urlEscaped: String {
-        return self.addingPercentEncoding(withAllowedCharacters: .urlHostAllowed)!
-    }
-}
+#if os(OSX)
+import AppKit
+#else
+import Foundation
```

Comment:
@BasThomas I came to that realization after awhile! I have to stop thinking in the extreme that everything not in UIKit is Foundation. I originally thought that Foundation imported CoreGraphics because they have an `Int.init(CGFloat)`... Stopping to think -- it's actually more likely that CoreGraphics extends Foundation with this initializer...  I'm such a noob 😆